src/App.tsx

```diff
@@ -1,26 +1,185 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import { getTodos, addTodo, deleteTodo, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { FilterStates } from './types/enums';
+import { TodoItem } from './components/TodoItem';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
```

Comment:
u can create `enum` for all ur error. Do it for all error
```suggestion
  const [error, setError] = useState<ErrorType>(ErrorType.DEFAULT);
```
```
export enum ErrorType = {
  DEFAULT: '',
  ERROR_TITLE: 'Error text........'
  ............
}
```

Reply:
Not fixed!